src/test/java/teammates/test/cases/testdriver/HtmlHelperTest.java

```diff
@@ -135,4 +135,20 @@ public void testConvertToStandardHtmlPart() throws Exception {
         
         //other cases are tested in testComparison
     }
+    
+    @Test
+    public void testReplacement() throws IOException {
```

Comment:
That wasn't the intention actually... this is meant to be the test for processes gone through while doing 'verifyHtml*`. `GodModeTest` is testing God mode, i.e regeneration. They have a lot of overlaps, but definitely not the same thing.